frontend/src/App.js

```diff
@@ -66,18 +62,36 @@ const router = createBrowserRouter([
           },
           {
             path: appRoutes.ADMINISTRATION_USERS,
-            element: <Paper elevation={5} sx={{ padding: "1rem", position: 'relative', minHeight: '80vh' }}>
-              <UserTabPanel />
-            </Paper>,
+            element: (
+              <Paper
+                elevation={5}
+                sx={{
+                  padding: '1rem',
+                  position: 'relative',
+                  minHeight: '80vh',
+                }}
+              >
+                <UserTabPanel />
+              </Paper>
+            ),
             // loader: data => data, // loader will pass data to useMatches
             handle: {
-              crumb: data => `Users`, // data from loader is passed into our crumb function so we can manipulate the output
+              crumb: () => `Users`, // data from loader is passed into our crumb function so we can manipulate the output
             },
           },
         ],
       },
     ],
   },
+  {
```

Comment:
edit users will not have breadcrumbs so it can stay here. the above route group is for layouts that need breadcrumbs as denoted by the `crumb` parameter for `<Layout/>`.

The route will change ( idir/bceid ) according to your other comment, we will add those routes here.